src/appdistribution/distribution.ts

```diff
@@ -23,7 +26,8 @@ export class Distribution {
     const distributionType = path.split(".").pop();
     if (
       distributionType !== DistributionFileType.IPA &&
-      distributionType !== DistributionFileType.APK
+      distributionType !== DistributionFileType.APK &&
+      distributionType !== DistributionFileType.AAB
     ) {
       throw new FirebaseError("unsupported distribution file format, should be .ipa or .apk");
```

Comment:
Update to "unsupported distribution file format, should be .ipa, .apk or .aab"

Reply:
Done!